caas/kubernetes/provider/application/application.go

```diff
@@ -1277,15 +1277,9 @@ func (a *app) applicationPodSpec(config caas.ApplicationConfig) (*corev1.PodSpec
 		ImagePullPolicy: corev1.PullIfNotPresent,
 		Image:           config.CharmBaseImagePath,
 		WorkingDir:      jujuDataDir,
-		Command:         []string{"/charm/bin/containeragent"},
-		Args: []string{
-			"unit",
-			"--data-dir", jujuDataDir,
-			"--charm-modified-version", strconv.Itoa(config.CharmModifiedVersion),
-			"--append-env", "PATH=$PATH:/charm/bin",
-			"--show-log",
-		},
-		Env: env,
+		Command:         []string{"/charm/bin/pebble"},
+		Args:            []string{"run"},
+		Env:             env,
 		SecurityContext: &corev1.SecurityContext{
 			RunAsUser:  pointer.Int64Ptr(0),
 			RunAsGroup: pointer.Int64Ptr(0),
```

Comment:
Do we need to change the liveness/readiness probes below now that we are running Pebble? Might make sense to use similar values to the workload containers

Reply:
i.e. should we keep the original agent health checks, or instead use the Pebble health checks